package.json

```diff
@@ -26,6 +26,7 @@
     "mongoose": "^5.9.10",
     "morgan": "^1.10.0",
     "nock": "^12.0.3",
+    "pify": "^5.0.0",
```

Comment:
Why is adding this dependency better than just using `util.promisify`?

Reply:
`pify` is faster and it supports multiple arguments returned by the callback.